zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -449,14 +542,37 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
             step.rw_indices[15 + rw_offset],
         ]
         .map(|idx| block.rws[idx].call_context_value());
-        let [caller_balance_pair, callee_balance_pair, (callee_nonce, _), (callee_code_hash, _)] =
-            [
-                step.rw_indices[16 + rw_offset],
+
+        // get caller balance
+        let (caller_balance, _) = block.rws[step.rw_indices[16 + rw_offset]].account_value_pair();
```

Comment:
Shouldn't the `rw_offset` be increased here?

Reply:
increased by `16 + rw_offset` , last rw indices is `15 + rw_offset`